src/app/common/grade-icon/grade-icon.component.scss

```diff
@@ -0,0 +1,20 @@
+.grade-icon {
```

Comment:
SELF NOTE: Implement using tailwind

Reply:
add the Tailwind classes to the html in this PR

Reply:
Using Tailwind now